README.md

```diff
@@ -6,6 +6,20 @@
 
 For documentation, visit [https://travix-international.github.io/frint](https://travix-international.github.io/frint).
 
+# Motivation
```

Comment:
isn't this duplicating https://github.com/Travix-International/frint/blob/master/docs/introduction/Motivation.md?